### ISSUE TITLE
Medial epicondyle avulsion fracture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10828]
**Preferred term label:**
Medial epicondyle avulsion fracture

**Synonyms**

**Definition (free text, please give PubMed ID)**
AP radiograph of the elbow showing avulsion fracture of the medial epicondyle

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Little league elbow

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
The term "[Avulsion fractured humerus HP:0041088](https://hpo.jax.org/browse/term/HP:0041088)" sounds more general, but as the Medial epicondyle is located on the humerus, I wonder if they are synonyms, i.e., if this is the main or only kind of Avulsion fractured humerus? The latter term was created a while back in an experiment with templates that was not really successful and needs a better definition anyway.
Thoughts?

**Comment by MickeySegal** [comment]
As to whether HP:0041088 should be changed to be specific to the medial epicondyle I will ask our orthopedist.

I see that there is also a medial epicondyle of the femur, so I changed our term to "humerus medial epicondyle avulsion fracture".

**Comment by MickeySegal** [comment]
The orthopedist replied:

I would specify the medial epicondyle avulsion fracture rather than keep it generalized. There can be avulsion fractures at several anatomic locations along the humerus so this will help clarify and allow for future data points to be created for avulsion fractures elsewhere in the humerus.

**Comment by pnrobinson** [comment]
@MickeySegal 
[Medial epicondyle apophysitis HP:6001263](https://hpo.jax.org/browse/term/HP:6001263) is distinct from the fracture term, but it also has the synonym "Little league elbow". From what I see, "Little league elbow" belongs more to the apophysitis than to the fracture term, correct? I am not adding the synonym for now, please reopen if this still needs work.


Adding new term: 
- ID: HP:6001436
- label: Medial humerus epicondyle avulsion fracture
- parents: [Abnormal humerus morphology (HP:0031095)](https://hpo.jax.org/app/browse/term/HP:0031095)
- pmid: PMID:37344531
- def: An avulsion fracture of the medial epicondyle of the elbow. Radiological examination in the anteroposterior and lateral projection can be used to make the diagnosis and to determine the degree and direction of the fracture displacement.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10828
- synonyms:

**Comment by MickeySegal** [comment]
Little league elbow does not connote a fracture.
